docs/traffic-policy/index.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Overview
+---
+
+import Link from "@docusaurus/Link";
+
+# Traffic Policy
+
+<div class="prose prose-gray dark:prose-invert mt-2 text-lg">
+	<p>
+		ngrok's Traffic Policy is a powerful configuration language for ngrok
```

Comment:
it feels strange to describe it as a configuration language. maybe skip it with something like `A Traffic Policy allows you to define how to filter, match, manage and orchestrate traffic on your endpoint or edge. It is a configuration that can be written in JSON or YAML.`